pgo/cmd/root.go

```diff
@@ -54,13 +55,17 @@ func init() {
 	GREEN = color.New(color.FgGreen).SprintFunc()
 	RED = color.New(color.FgRed).SprintFunc()
 
+	// Go currently guarantees an error when attempting to load OS_TRUST for
+	// windows-based systems (see crypto/x509/cert_pool.go)
```

Comment:
I'd like a link to `https://golang.org/issue/16736` rather than the `crypto` files here.

Reply:
Agreed, updated